src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+a {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav__item {
+  list-style-type: none;
+  display: flex;
+  height: 60px;
+}
+
+.nav__list {
+  width: 760px;
```

Comment:
try to set `margin` or `gap` between list items, instead of hardcoding its `width` 😉